src/Signal/Time.purs

```diff
@@ -34,7 +34,8 @@ every = runFn3 everyP constant now
 -- |Returns the number of milliseconds since an arbitrary, but constant, time in the past.
 foreign import now """
   function now() {
-    var perf = typeof performance !== 'undefined' ? performance : null;
+    var perf = typeof performance !== 'undefined' ? performance : null,
+        process = typeof process !== 'undefined' ? process : null;
```

Comment:
<code>process</code> will not be <code>null</code> in nodejs, it's part of node's global object